### PR TITLE
[frontend][api][SIM-#120] Implement Active/Inactive filter on Interpreter Dir

### PR DESCRIPTION
**Pull request by fwpushan**
------------------------------------------------------------------------------
1. Add new Active/Inactive options interpreter directory
2. Update Search DB query in API
-----

### CODE REVIEW

File: api/src/interpreter/dto/paginate-interpreter-query.dto.ts [hunk 1]
```diff
@@ -79,12 +79,20 @@ export class PaginateInterpreterQueryDto extends PaginationQueryDTO {
   @IsOptional()
   keywords?: string;
 
+  @ApiProperty({
+    description: 'Interpreter Active/Inactive indicator',
+    example: faker.random.boolean(),
+  })
+  @IsOptional()
+  active?: string;
```
Comment (michel-fresh) [inline api/src/interpreter/dto/paginate-interpreter-query.dto.ts:87]: not type boolean?

File: frontend/src/views/InterpreterDirectory.tsx [hunk 2]
```diff
@@ -30,10 +31,15 @@ export default function InterpreterDirectory () {
 
   const getSearchResults = async (params: InterpreterSearchParams) => {
     setSearch(params)
+    // Changing query options as per user selection
+    const query: InterpreterSearchParams = { 
+      ... params,
```
Comment (michel-fresh) [inline frontend/src/views/InterpreterDirectory.tsx:36]: there is a space here, does that compile?

File: frontend/src/views/InterpreterDirectory.tsx [hunk 3]
```diff
@@ -30,10 +31,15 @@ export default function InterpreterDirectory () {
...
+      active: params.active === 'Active' ? 'TRUE' : 'FALSE',
```
Comment (michel-fresh) [inline frontend/src/views/InterpreterDirectory.tsx:37]: Why not pass these as `true` or `false`?

**Review by michel-fresh — CHANGES_REQUESTED** [review]

File: api/src/interpreter/dto/paginate-interpreter-query.dto.ts (hunk 1 above)
Comment (fwpushan) [reply api/src/interpreter/dto/paginate-interpreter-query.dto.ts:87]: Raw query generator utility unable to generate query for boolean value (specially for case false), but works fine if I supply this as string.

File: frontend/src/views/InterpreterDirectory.tsx (hunk 3 above)
Comment (fwpushan) [reply frontend/src/views/InterpreterDirectory.tsx:37]: Raw query generator utility unable to generate query for boolean value (specially for case false), but works fine if I supply this as string.

**Review by michel-fresh — APPROVED** [review]
nice 👍

**Pull request merged**